packages/strapi-connector-mongoose/lib/mount-models.js

```diff
@@ -27,6 +27,18 @@ module.exports = ({ models, target }, ctx) => {
       primaryKeyType: 'string',
     });
 
+    if (!definition.uid.startsWith('strapi::') && definition.modelType !== 'component') {
+      definition.attributes['created_by'] = {
```

Comment:
Should these be using underscores? The `updatedAt` and `createdAt` are what is normally used on mongoose so should we also use `createdBy` and `updatedBy` for mongoose and keep the underscores for SQL? (Honestly not sure why the two are different for different ORMs but 🤷 )

Reply:
For simplicity's sake we are using one unique name everywhere for this.

Reply:
Good to know, wondering if in the future we should look at changing the mongoose timestamp fields to match 🤔